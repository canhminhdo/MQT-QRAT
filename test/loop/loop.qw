set show timing off .
set random seed 0 .

prog PROG_LOOP is
var q : qubit;
init
    q := random;
begin
    q := H[q];
    while M[q] = 1 do q := H[q]; od;
end

search in PROG_LOOP with =>! such that true .

quit .