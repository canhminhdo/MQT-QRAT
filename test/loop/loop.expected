Show timing is off.
Random seed is set to 0.
==========================================
prog PROG_LOOP
==========================================
search in PROG_LOOP with =>! such that true .

Solution 1 (state 2)
states: 3
probability: 0.501, quantum state: 
0: (0.754,-0.657)

No more solutions.
states: 5
Bye.
